src/app/inbox/components/notification-permission-institutional-connection/notification-permission-institutional-connection.component.ts

```diff
@@ -26,6 +26,6 @@ export class NotificationPermissionInstitutionalConnectionComponent
   ngOnInit(): void {}
 
   navigateTo(val) {
-    this.window.location.href = val
+    this.window.open(val, '_blank')
```

Comment:
`noopener` `noreferrer` needed

Reply:
(for the other JS `open` as ell 

More on this here: 
https://developer.mozilla.org/en-US/docs/Web/API/Window/open#window_functionality_features